uint16 Num
raw4cam[] raw_data